src/quo/components/inputs/input/view.cljs

```diff
@@ -7,8 +7,8 @@
             [react-native.core :as rn]
             [react-native.platform :as platform]))
 
-(defn- label-&-counter
-  [{:keys [label current-chars char-limit variant-colors theme]}]
+(defn- label-line
```

Comment:
In designs input field has labels at top-left and top-right sides. In our component we had only characters counter in the right side, so i added a right-side label.